### PR TITLE
Add Sentinel 2 grayscale layers

### PR DESCRIPTION
**Pull request by GypsyBojangles**
Add Sentinel 2 grayscale layers

### CODE REVIEW

File: dev/services/wms/ows/wms_cfg.py [hunk 1]
```diff
@@ -1811,6 +1811,193 @@
...
+                            },
+                            "blue": {
+                                "nbart_coastal_aerosol": 1.0
+                            }
+                        },
+                        "scale_range": [0.0, 3000.0]
+                    },
+                    {
+                        "name": "blue",
+                        "title": "Blue - 440",
```
Comment (roarmstrong) [inline dev/services/wms/ows/wms_cfg.py]: Is 440 the correct centre wavelength for this style?

File: dev/services/wms/ows/wms_cfg.py [hunk 2]
```diff
@@ -1995,6 +2182,193 @@
...
```
Comment (roarmstrong) [inline dev/services/wms/ows/wms_cfg.py]: Band Centre?

File: dev/services/wms/ows/wms_cfg.py [hunk 3]
```diff
@@ -1811,6 +1811,193 @@
...
+                            },
+                            "blue": {
+                                "nbart_nir_1": 1.0
+                            }
+                        },
+                        "scale_range": [0.0, 3000.0]
+                    },
+                    {
+                        "name": "narrow_nir",
+                        "title": "Narrow Near Infrared - 870",
```
Comment (roarmstrong) [inline dev/services/wms/ows/wms_cfg.py]: Band Centre?

File: dev/services/wms/ows/wms_cfg.py [hunk 4]
```diff
@@ -1995,6 +2182,193 @@
...
```
Comment (roarmstrong) [inline dev/services/wms/ows/wms_cfg.py]: Band Centre

File: dev/services/wms/ows/wms_cfg.py (hunk 3 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows/wms_cfg.py]: Product owner want's this rounded to the nearest 10nm

File: dev/services/wms/ows/wms_cfg.py (hunk 4 above)
Comment (GypsyBojangles) [reply dev/services/wms/ows/wms_cfg.py]: Product owner want's this rounded to the nearest 10nm

**Pull request merged**